reg/drone/physics/kinematics/Angular1D.0.1.uavcan

```diff
@@ -0,0 +1,5 @@
+# Rotation about an axis.
```

Comment:
I think this should be separated by type (i.e, `kinematics/position/Angular1D.0.1.uavcan`). In very few situations you are getting all the kinematic data types from the same source.

Reply:
For separate data, we have the SI namespace: `uavcan.si.unit.length.Scalar.1.0` and so on. This type (and its siblings) is specifically designed to provide a combined set of kinematic states in one entity.